MpepcPlannerROS:
  potential_topic: "/move_base/NavfnROSExt/potential" # Potential topic
  cost_theta: 0.05          # Theta gradient cost
  cost_collision: 1.0       # Collision cost
  cost_v: 0.05              # Velocity cost
  cost_w: 0.05              # Angular cost
  cost_collision_sigma: 0.2 # SIGMA